src/main/java/soundCloud/ui/pages/LoginPage.java

```diff
@@ -0,0 +1,77 @@
+/*
```

Comment:
I guess the "soundcloud" directory is not part of your testing project anymore.

Reply:
done